app/pages/cif/projects.tsx

```diff
@@ -88,18 +100,22 @@ export function Projects({ preloadedQuery }: RelayProps<{}, projectsQuery>) {
         orderByPrefix: "OPERATOR_BY_OPERATOR_ID__TRADE_NAME",
       }),
       new TextFilter("Proposal Reference", "proposalReference"),
-      new TextFilter("Status", "status", {
-        orderByPrefix: "PROJECT_STATUS_BY_PROJECT_STATUS_ID__NAME",
-      }),
+      new SearchableDropdownFilter(
+        "Status",
+        "status",
+        allProjectStatuses.edges.map((e) => e.node.name),
+        { hideClearButton: true }
```

Comment:
This is the default now, we can remove it

Reply:
removed

Reply:
Removed